mount.go

```diff
@@ -187,9 +188,21 @@ type storageHandler func(storage pb.Storage, s *sandbox) (string, error)
 
 // storageHandlerList lists the supported drivers.
 var storageHandlerList = map[string]storageHandler{
-	driver9pType:   virtio9pStorageHandler,
-	driverBlkType:  virtioBlkStorageHandler,
-	driverSCSIType: virtioSCSIStorageHandler,
+	driver9pType:        virtio9pStorageHandler,
+	driverBlkType:       virtioBlkStorageHandler,
+	driverSCSIType:      virtioSCSIStorageHandler,
+	driverEphemeralType: ephemeralStorageHandler,
+}
+
+func ephemeralStorageHandler(storage pb.Storage, s *sandbox) (string, error) {
+	if _, err := os.Stat(storage.MountPoint); os.IsNotExist(err) {
+		os.MkdirAll(storage.MountPoint, os.ModePerm)
+		_, err = commonStorageHandler(storage)
+		return "", err
+
+	} else {
```

Comment:
You can delete this else to make CI happy.

Reply:
sure.